ludwig/schema/trainer.py

```diff
@@ -102,21 +91,12 @@ class BaseTrainerConfig(schema_utils.BaseMarshmallowConfig, ABC):
     )
 
 
-@register_trainer_schema("trainer")
+@register_trainer_schema("ecd_ray_legacy")
```

Comment:
There's two registrations here. Just checking -- is that intentional?

Reply:
Yes it is, thanks for checking! Basically this allows the `ray_legacy_trainer` to be accessed by the user. Since it uses the same schema as the regular trainer, I just registered it twice here.